### PR TITLE
Changed make tag

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Changes `make tag` to be what I run every time

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jaxesn — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/573#pullrequestreview-762615535" title="LGTM">jaxesn</a>*, *<a href="https://github.com/aws/eks-distro/pull/573#" title="Author self-approved">kschumy</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [jaxesn,kschumy]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
